src/Command/GmvCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\GmvBundle\Command;
+
+use Sylius\GmvBundle\Parser\DateParserInterface;
+use Sylius\GmvBundle\Provider\GmvProviderInterface;
+use Sylius\GmvBundle\Validator\InputParametersValidatorInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webmozart\Assert\Assert;
+
+#[AsCommand(
+    name: 'sylius:gmv:calculate',
+    description: 'Lightweight local command to calculate the Sylius instance GMV within a specific period',
+)]
+final class GmvCommand extends Command
+{
+    public function __construct(
+        private readonly InputParametersValidatorInterface $validator,
+        private readonly DateParserInterface $dateParser,
+        private readonly GmvProviderInterface $gmvProvider,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument(
+                'periodStart',
+                InputArgument::OPTIONAL,
+                'The start of the period (e.g., 05/2024)',
+                $this->dateParser->getDefaultStartDate()->format('m/Y'),
+            )
+            ->addArgument(
+                'periodEnd',
+                InputArgument::OPTIONAL,
+                'The end of the period (e.g., 06/2024)',
+                $this->dateParser->getDefaultEndDate()->format('m/Y'),
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $periodStart = $input->getArgument('periodStart');
+        $periodEnd = $input->getArgument('periodEnd');
+
+        Assert::string($periodStart);
+        Assert::string($periodEnd);
```

Comment:
Maybe check if they're both strings and handle it similarly to the case below, instead of throwing exceptions